### PR TITLE
[Doc] Update pulsar-client cli doc with newer options.

### PR DESCRIPTION
**Pull request by MarvinCai**

### CODE REVIEW

File: site2/docs/reference-cli-tools.md [hunk 1]
```diff
@@ -315,6 +315,10 @@ Options
 |`-m`, `--messages`|Comma-separated string of messages to send; either -m or -f must be specified|[]|
 |`-n`, `--num-produce`|The number of times to send the message(s); the count of messages/files * num-produce should be below 1000|1|
 |`-r`, `--rate`|Rate (in messages per second) at which to produce; a value 0 means to produce messages as fast as possible|0.0|
+|`-c`, `--chunking`|Should split the message and publish in chunks if message size is larger than allowed max size|false|
```
Comment (Jennifer88huang) [inline site2/docs/reference-cli-tools.md:318]:
```suggestion
|`-c`, `--chunking`|Split the message and publish in chunks if the message size is larger than the allowed max size|false|
```

File: site2/docs/reference-cli-tools.md [hunk 2]
```diff
@@ -315,6 +315,10 @@ Options
...
+|`-s`, `--separator`|Character to split messages string on default is comma|","|
+|`-k`, `--key`|Message key to add|key=value string, like k1=v1,k2=v2.|
+|`-p`, `--properties`|Properties to add, Comma separated| |
```
Comment (Jennifer88huang) [inline site2/docs/reference-cli-tools.md:321]:
```suggestion
|`-p`, `--properties`|Properties to add, comma separated| |
```

File: site2/docs/reference-cli-tools.md [hunk 3]
```diff
@@ -335,7 +339,10 @@ Options
 |`-s`, `--subscription-name`|Subscription name||
 |`-t`, `--subscription-type`|The type of the subscription. Possible values: Exclusive, Shared, Failover, Key_Shared.|Exclusive|
 |`-p`, `--subscription-position`|The position of the subscription. Possible values: Latest, Earliest.|Latest|
-
+|`-m`, `--subscription-mode`|Subscription mode.|Durable|
+|`-q`, `--queue-size`|Consumer receiver queue size.|0|
```
Comment (Jennifer88huang) [inline site2/docs/reference-cli-tools.md:343]: do you mean "The queue size of the consumer receiver"?

File: site2/docs/reference-cli-tools.md [hunk 4]
```diff
@@ -335,7 +339,10 @@ Options
...
+|`-mc`, `--max_chunked_msg`|Max pending chunk messages.|0|
+|`-ac`, `--auto_ack_chunk_q_full`|Auto ack for oldest message on queue is full.|false|
```
Comment (Jennifer88huang) [inline site2/docs/reference-cli-tools.md:345]:
```suggestion
|`-ac`, `--auto_ack_chunk_q_full`|Auto ack for the oldest message on queue is full.|false|
```

**Review by Jennifer88huang — COMMENTED** [review]
@MarvinCai thank you very much for adding the content. Does it apply only to 2.7.1 and 2.8.0?

File: site2/docs/reference-cli-tools.md [hunk 5]
```diff
@@ -315,6 +315,10 @@ Options
 |`-m`, `--messages`|Comma-separated string of messages to send; either -m or -f must be specified|[]|
 |`-n`, `--num-produce`|The number of times to send the message(s); the count of messages/files * num-produce should be below 1000|1|
 |`-r`, `--rate`|Rate (in messages per second) at which to produce; a value 0 means to produce messages as fast as possible|0.0|
+|`-c`, `--chunking`|Should split the message and publish in chunks if message size is larger than allowed max size|false|
+|`-s`, `--separator`|Character to split messages string on default is comma|","|
```
Comment (Anonymitaet) [inline site2/docs/reference-cli-tools.md:319]: What does "Character to split messages string on default is comma" mean?

File: site2/docs/reference-cli-tools.md (hunk 2 above)
Comment (Anonymitaet) [inline site2/docs/reference-cli-tools.md:321]: Do you mean "Properties to add. If you want to add multiple properties, use the comma as the separator."?

**Comment by MarvinCai** [comment]
> @MarvinCai thank you very much for adding the content. Does it apply only to 2.7.1 and 2.8.0?

We haven't update the doc for this client cli for a while I think, I'm not sure when did each options added to the client cli, so I only update 2.7.1 which is the version we're sure that all options are available.

File: site2/docs/reference-cli-tools.md (hunk 3 above)
Comment (MarvinCai) [reply site2/docs/reference-cli-tools.md:343]: changed to `The size of consumer's receiver queue`

File: site2/docs/reference-cli-tools.md (hunk 5 above)
Comment (MarvinCai) [reply site2/docs/reference-cli-tools.md:319]: missing a comma I think, should be `Character to split messages string on, default is comma`

**Review by Jennifer88huang — APPROVED** [review]

**Review by sijie — APPROVED** [review]

**Pull request merged**